layouts/partials/components/etro.html

```diff
@@ -0,0 +1,6 @@
+<div class="h-96">
+  <iframe class="w-full h-full"
+    title="Etro Gearset" 
+    src="https://etro.gg/embed/gearset/{{ .uuid }}">
```

Comment:
since we're only dealing with one value here, instead of creating a dictionary just so we can have a `.uuid` field, maybe it's appropriate to *only* pass in `$uuid` and use it here as just `.`?

Reply:
Yeah I originally had more parameters being passed, but removed them and forgot to remove the dict.